### ISSUE TITLE
error downloading clip with download command in 1.13.0

### ISSUE THREAD

**Issue by tpreischadt** [ihabunek/twitch-dl#45]
Downloading the following clip:

[FuriousResilientWrenchAsianGlow](https://clips.twitch.tv/FuriousResilientWrenchAsianGlow)

with the following command:

```sh
./twitch-dl.1.13.0.pyz download -q source https://clips.twitch.tv/FuriousResilientWrenchAsianGlow
```

Yielded the following output:

```sh
Looking up clip...
Found: GAU PEIDAO by Gaules, playing Among Us (28 sec)
Selected URL: https://clips-media-assets2.twitch.tv/39698797725-offset-38352.mp4
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "./twitch-dl.1.13.0.pyz/__main__.py", line 3, in <module>
  File "./twitch-dl.1.13.0.pyz/twitchdl/console.py", line 223, in main
  File "./twitch-dl.1.13.0.pyz/twitchdl/commands.py", line 275, in download
  File "./twitch-dl.1.13.0.pyz/twitchdl/commands.py", line 325, in _download_clip
  File "./twitch-dl.1.13.0.pyz/twitchdl/commands.py", line 211, in _clip_target_filename
KeyError: 'createdAt'
```

Clip was found but there is no createdAt key in the clip dict.

**Comment by tpreischadt** [comment]
Trying to figure out the error but I can't seem to build from source =(

**Comment by tpreischadt** [comment]
Managed to alter source code and re-compile it and clip dict is different. As expected, has no `createdAt` attribute.

[Output](https://i.imgur.com/9nZQ3Wb.jpg)

**Comment by ihabunek** [comment]
For some reason several fields were missing from the clip query. Fixed in 1.13.1.
